src/state/data-types/actions.js

```diff
@@ -23,7 +39,25 @@ export const fetchDataTypes = (page = { page: 1, pageSize: 10 }, params = '') =>
           dispatch(setDataTypes(json.payload));
           dispatch(toggleLoading('dataTypes'));
           dispatch(setPage(json.metaData));
-        } else if (json && json.errors) {
+        } else {
+          dispatch(addErrors(json.errors.map(err => err.message)));
+          dispatch(toggleLoading('dataTypes'));
```

Comment:
Please remove the two `dispatch(toggleLoading('dataTypes'));` calls from the if/else branches, and make a single call outside the if/else